zoom/zoom_rapid7_plugin.egg-info/PKG-INFO

```diff
@@ -0,0 +1,10 @@
+Metadata-Version: 1.0
```

Comment:
I am guessing these zoom related files are not a part of this PR!

Reply:
Weird...WTF.

Reply:
Should be removed.